docs/HowTo/clusterconnectivity/aiven-kafka-cluster-sasl-ssl-protocol.md

```diff
@@ -48,7 +48,7 @@ Kafka® and Klaw using SSL protocol:
    icon that is available on the right-hand side of each cluster
    row.
 9. Open the `application.properties` file located in the
-   [klaw/cluster-api/src/main/resources] directory.
+   [Klaw/cluster-api/src/main/resources] directory.
```

Comment:
```suggestion
   `klaw/cluster-api/src/main/resources` directory.
```

Hi! Your change actually surfaced a different error in our documentation. 

Since this is a file reference, it should not be in `[]` brackets. 

It looks like this in on the site (which is wrong):

<img width="857" alt="Screenshot 2023-09-29 at 12 22 30" src="https://github.com/Aiven-Open/klaw-docs/assets/943800/c2e9ce8a-428d-470e-9e0b-65df1e5b1453">

Could you update that for us? 

The filename should be in backticks instead of brackets. And since it's a file name, "Klaw" is not capitalised. (see suggestion)